### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nighthacker**
hi could you please take a look @probkakodu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # nighthacks
 Bla bla bla
 nowsza awersja
+
+dodawanie uprawnien
```
Comment (probkakodu) [inline README.md]: strange very strange

**Comment by nighthacker** [comment]
any advise how to solve it ? @probkakodu

**Comment by probkakodu** [comment]
hmm ok

File: README.md (hunk 1 above)
Comment (nighthacker) [reply README.md]: but why

**Comment by probkakodu** [comment]
ok

**Pull request merged**